src/goto-instrument/contracts/utils.cpp

```diff
@@ -242,3 +242,30 @@ bool is_assigns_clause_replacement_tracking_comment(const irep_idt &comment)
   return id2string(comment).find(ASSIGNS_CLAUSE_REPLACEMENT_TRACKING) !=
          std::string::npos;
 }
+
+assignst widen_assigns(const assignst &assigns)
```

Comment:
Should this perhaps take a non-const reference if the usage pattern always (as is the case at the moment) is `x = widen_assigns(x);`?

Reply:
Modified the function to a `void` function that updates its non-const reference argument.